### PR TITLE
Remove logging in ContosoCookbook and update Logging-Loggly

### LINKED ISSUE #116: [Update] ContosoCookbook - Prism.Logging
### Sample

Contoso Cookbook... maybe more???

#### Reason for Update

As was noticed from this Tweet

https://twitter.com/jacobegner/status/1358104406995177477

Prism.Logging has been removed in Prism 8. We need to verify that it is removed from any relevant samples.

### PR DESCRIPTION
**Pull request by Adam--**
Building sample-apps/ConsotoCookbook and advanced-topics/Logging-Loggly was failing due to a update to Prism 8 and the removal of ILoggerFacade.

In ConsotoCookbook, logging was removed.

In Logging-Loggly, Prism.Plugin.Logging.Loggly was updated.

Closes #116

### CODE REVIEW

**Comment by brianlagunas** [comment]
I would actually prefer to just remove logging from this sample altogether. It's providing no value

**Comment by Adam--** [comment]
Agreed and removed.

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs [hunk 1]
```diff
@@ -43,7 +44,7 @@ protected override void RegisterTypes(IContainerRegistry containerRegistry)
 
             // TODO: Be sure to get your Loggly Consumer Token. Paste it here to avoid typing it...
```
Comment (brianlagunas) [inline advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:45]: remove this

File: sample-apps/ContosoCookbook/ContosoCookbook/ContosoCookbook/ContosoCookbook.csproj [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="Prism.Plugin.Logging.Abstractions" />
```
Comment (brianlagunas) [inline sample-apps/ContosoCookbook/ContosoCookbook/ContosoCookbook/ContosoCookbook.csproj:10]: remove this

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs [hunk 3]
```diff
@@ -7,6 +7,7 @@
 using PrismSample.ViewModels;
 using PrismSample.Views;
 using Xamarin.Forms;
+using LogglyOptions = PrismSample.Logging.LogglyOptions;
```
Comment (brianlagunas) [inline advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: remove this

File: Packages.props [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <PackageReference Update="Prism.Unity.Forms" Version="$(PrismVersion)" />
     <PackageReference Update="Prism.DryIoc.Forms" Version="$(PrismVersion)" />
     <PackageReference Update="Prism.Plugin.Popups" Version="7.2.0.1046" />
-    <PackageReference Update="Prism.Plugin.Logging.Loggly" Version="7.2.0.1114" />
+    <PackageReference Update="Prism.Plugin.Logging.Loggly" Version="8.0.11-beta" />
```
Comment (brianlagunas) [inline Packages.props:13]: remove this

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 3 above)
Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build. 
This fixes an ambiguous reference between PrismSample.Logging.LogglyOptions and Prism.Logging.Loggly.LogglyOptions that was introduce by Prism.Logging.Loggly exposing a LogglyOptions. I don't really know much about Prism.Logging.Loggly so I don't know why it now exposes a LogglyOptions, but in the readme it says to implement ILogglyOptions and register it (https://github.com/dansiegel/Prism.Plugin.Logging#loggly).

I could rename the sample's LogglyOptions or I could create a new Prism.Logging.Loggly LogglyOptions, configure it, and register it instead. What do you think?

File: Packages.props (hunk 4 above)
Comment (Adam--) [reply Packages.props:13]: Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build.
It was failing due to a update to Prism 8. The version of Prism.Plugin.Logging.Loggly that was referenced expected an ILoggerFacade.

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 3 above)
Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build.
This fixes an ambiguous reference between PrismSample.Logging.LogglyOptions and Prism.Logging.Loggly.LogglyOptions that was introduce by Prism.Logging.Loggly exposing a LogglyOptions. I don't really know much about Prism.Logging.Loggly so I don't know why it now exposes a LogglyOptions, but in the readme it says to implement ILogglyOptions and register it (https://github.com/dansiegel/Prism.Plugin.Logging#loggly).

I could rename the sample's LogglyOptions or I could create a new Prism.Logging.Loggly LogglyOptions, configure it, and register it instead. What do you think?

File: sample-apps/ContosoCookbook/ContosoCookbook/ContosoCookbook/ContosoCookbook.csproj (hunk 2 above)
Comment (Adam--) [reply sample-apps/ContosoCookbook/ContosoCookbook/ContosoCookbook/ContosoCookbook.csproj]: Good catch, removed

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 3 above)
Comment (brianlagunas) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: oh, sorry, I should have paid more attention. This is not the contoso sample. I though all changes were made to a single  sample. Disregard

File: Packages.props (hunk 4 above)
Comment (brianlagunas) [reply Packages.props:13]: oh, sorry, I should have paid more attention. This is not the contoso sample. I though all changes were made to a single  sample. Disregard

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 1 above)
Comment (brianlagunas) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:45]: ignore this one, it's not the contoso sample I thought we were dealing with.

**Review by brianlagunas — APPROVED** [review]

**Comment by brianlagunas** [comment]
Thanks for the PR

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 3 above)
Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: I removed the sample's LogglyOptions and am now registering a configured instance of the plugin's LogglyOptions. I believe this is how it's intended to be used.

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 1 above)
Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:45]: Removed.

Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:45]: Didn't see your comment (it wasn't showing at all) before posting my last comment in the conversation. Sorry about that.
I did end up removing it after I looked over Prism.Plugins.Logging.Loggly and determined it was probably best to register a new instance of LogglyOptions. It looks like you merged but didn't get those changes though.

File: advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs (hunk 3 above)
Comment (Adam--) [reply advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs:10]: Didn't see your comment (it wasn't showing at all) before posting my last comment in the conversation. Sorry about that.
It looks like you merged but didn't get those changes though.

**Comment by Adam--** [comment]
No problem glad to help even if it is just a little bit.

**Pull request merged**